RFCs/first-person-mercator-viewport-rfc.md

```diff
@@ -0,0 +1,233 @@
+
+* **Camera Coordinates** - Our hybrid applications usually use data with coordinates in `METER_OFFSET`s, transformed by modelMatrices. Being able to position camera(s) in the same ways (relative coordinates + matrices) is critical as this would dramatically simplify applications by allowing them to set camera positions inside existing objects simply by copying their coordinates.
+
+* **FOV sync** - Since our 3D geometry is completely general, we have the ability to show it from any camera position, height, and direction (including dual cameras for stereoscopic view etc).
+However, to synchronize our 3D data with external perspective-enabled map systems like Mapbox GL, we need to adjust our 3D camera to follow the map’s when in “hybrid” map/3D mode.
+The map system typically locks in FoV (Field of view), viewing angle (always pitched downward) and altitude (camera height over the “ground”).
+When rendering 3D environments on top of pre-rendered video (e.g. overlaying perception data on top of vehicle log cameras)
+
+
+## Proposal: Support for Cartographic Projection in all Viewports
+
+In deck.gl 4.1 only the WebMercatorViewport can handle layers with geospatial coordinates, as it is the only viewport that can produce the uniforms required by the project module.
+
+By moving some of the required properties into the base viewport, it would be possible to extend cartographic projection to all cameras.
+
+Note: There are subtleties around the positioning of the camera which is handled in the next proposal.
+
+
+### Viewport
+
+The current base viewport class would be extended with a lnglat property
+
+- lnglat anchor
+- zoom scale
+- offset
+
+This means that any viewport can calculate
+- projection center
+- distance scales
+which are two critical uniforms needed by the `project` shader module when handling geospatial coordinates.
+
+
+## Proposal: An Alternative Viewport Hierarchy
+
+In deck.gl v4.0, a viewport class hierarchy was introduced to support non-geospatial viewports. It separates between Perspective and Orthographic cameras (inspired by common WebGL frameworks).
+
+In retrospect this separation adds little value as it captures a trivial part of the camera (the projection matrix), while leaving the more difficult problems unsolved, such as control of camera position and direction. Such aspects are important, particularly for the planned animation extensions to deck.gl.
+
+As an alternative to the `Perspective`/`Orthographic` camera class, a `FirstPerson`/`ThirdPerson` approach.
+
+
+### FirstPersonViewport
+
+extends `Viewport`
+
+- player position (lngLat anchor + offset)
+- player direction
```

Comment:
is _player direction_ here referring to the _up_ vector (whether the player is standing straight) or the _face_ vector (the direction (s)he is looking at? 

also, we might want users to override the fov via the 1st person viewport as well.